### PR TITLE
Reduce false positives in pre-flight accel bias check

### PR DESCRIPTION
**Pull request by priseborough**
Addresses issue https://github.com/PX4/Firmware/issues/10833 

This PR adjusts the test threshold so that the threshold is scaled up with increasing uncertainty in state estimate. Eventually given sufficient acceleration/rotation all delta velocity bias variances collapse to 5E-8, eg:

screen shot 2019-01-31 at 8 51 26 am

which equates to an effective test threshold of   1.73E-3 + 3*sqrt(5E-8) = 2.4E-3 which is the same as the previous threshold and equates to an accel bias of 2.4E-3 / dtEKF = 2.4E-3 / 8E-3 = 0.3 m/s/s

Note that the EKF internally limits the max bias estimate to within an equivalent +-0.4 m/s/s, so we need this check to stay inside that range.

The horizontal axes are poorly observable when on ground and the variances remain high until the vehicle is moved/tilted. In the example shown below the XY axes stay at 100E-8 which would result in a test threshold of (1.73E-3 + 3*sqrt(100E-8)) / 8E-3 = 0.59 m/s/s which would not trigger.

screen shot 2019-01-31 at 9 10 35 am

**Test data / coverage**
Request for testing from affected users in https://github.com/PX4/Firmware/issues/10833 .

### CODE REVIEW

File: src/modules/commander/PreflightCheck.cpp [hunk 1]
```diff
@@ -576,6 +576,19 @@ static bool ekf2Check(orb_advert_t *mavlink_log_pub, vehicle_status_s &vehicle_s
...
+			if (report_fail) {
+				mavlink_log_critical(mavlink_log_pub, "Preflight Fail: High Accelerometer Bias");
+			}
+
+			success = false;
+			goto out;
+		}
+	}
 
 	if (fabsf(status.states[13]) > test_limit || fabsf(status.states[14]) > test_limit
```
Comment (dagar) [inline src/modules/commander/PreflightCheck.cpp]: Remove the original "High Accelerometer Bias" check?

Comment (priseborough) [reply src/modules/commander/PreflightCheck.cpp]: Reporting the reason for failure if report_fail is true is consistent with the way other test failures are handled.

Comment (dagar) [reply src/modules/commander/PreflightCheck.cpp]: Isn't this new "High Accelerometer Bias" check with uncertainty a replacement for the existing "High Accelerometer Bias"? The existing check is still going to trigger, and hence not reduce the number of false positives.

Comment (priseborough) [reply src/modules/commander/PreflightCheck.cpp]: Will fix.

**Pull request merged**